rebar.config

```diff
@@ -32,7 +32,7 @@
 
 
 {profiles, [
-            {test, [{deps, [{meck, "0.8.13"}]}]}
+            {test, [{deps, [{meck, "0.9.0"}]}, {erl_opts, [warnings_as_errors]}]}
```

Comment:
Note—I think you also need to fix this in the [`rebar.config.script`](https://github.com/erlcloud/erlcloud/blob/afe0067c864c72bf59b3309a61ba72730398a23b/rebar.config.script#L8) if you want rebar2 to use it.

Reply:
> want rebar2 to use it

I don't :D but I understand you might. I'll get on it.

Reply:
Pushed in 15984c6